### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Queues_ViewMessage_Screen reader : NVDA does not read text inside "Message text:" read only edit field 

### ISSUE THREAD

**Issue by pranshu512** [microsoft/AzureStorageExplorer#3465]
GitHubTags: #C&AI_Aug2020_BMAzure Tools Storage Explorer;#A11yMAS;#A11YTCS;#DesktopApp;#Azure Tools Storage Explorer;#WCAG1.3.1;#Win32;#A11ySev2;#Benchmark;#NVDA;#JAWS;

### **Environment Details:**
Application Name: Azure Tool Storage Explorer
Application version: 1.14.2
Windows Version - Windows10
Screen reader: NVDA Version:2020.2,JAWS(2020.2006.12ILM)

### **Repro Steps:**

1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Queues" of the selected subscription and expand it.
5. Navigate to any queue and click on it..
6. Navigate to View message button and click.
7. Start screen reader as NVDA and check whether NVDA read text inside "Message text:" read only edit field or not  .

### **Actual:**
NVDA does not read text inside "Message text:" read only edit field 

**Similar issue also observed with JAWS**

### **Expected:**
NVDA should read text inside "Message text:" read only edit field 

### **User Impact:**
The screen reader users will not get content inside "Message text:" read only edit field .

### **Attachment**

https://microsoftapc-my.sharepoint.com/:v:/g/personal/v-prraha_microsoft_com/EZvBcR9WlCxKogZrMr7OBE0Bi5InUK7xSwHCmyOk3Z8FhA?e=2c8eH5

**Comment by JasonYeMSFT** [comment]
@pranshu512 I have tried it and cannot reproduce it with NVDA 2020.2 and Storage Explorer 1.15.1. Could you double check if it is reproducible?

**Comment by pranshu512** [comment]
@JasonYeMSFT I tested this issue with NVDA 2020.2 and Storage Explorer 1.16.0 and it not reproducing.Hence,Closing this bug.
Refer attachment
[3465_Closed.zip](https://github.com/microsoft/AzureStorageExplorer/files/5346012/3465_Closed.zip)

**Comment by 12-shweta** [comment]
Github Tags:#Closed;
